src/Tribe/Attendees.php

```diff
@@ -935,4 +938,45 @@ public function update_attendee( $attendee, $attendee_data ) {
 		return $provider->update_attendee( $attendee, $attendee_data );
 	}
 
+	/**
+	 * Generate the export URL for exporting attendees
```

Comment:
The docblock must have a full stop
```suggestion
	 * Generate the export URL for exporting attendees.
```